card/db/schema.rb

```diff
@@ -9,121 +9,120 @@
 # from scratch. The latter is a flawed and unsustainable approach (the more migrations
 # you'll amass, the slower it'll run and the greater likelihood for issues).
 #
-# It's strongly recommended that you check this file into your version control system.
+# It's strongly recommended to check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20151105225559) do
+ActiveRecord::Schema.define(:version => 20141216053032) do
 
-  create_table "card_actions", force: :cascade do |t|
-    t.integer "card_id",         limit: 4
-    t.integer "card_act_id",     limit: 4
-    t.integer "super_action_id", limit: 4
-    t.integer "action_type",     limit: 4
+  create_table "card_actions", :force => true do |t|
+    t.integer "card_id"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.